dask_jobqueue/pbs.py

```diff
@@ -95,6 +95,10 @@ def __init__(self,
 
         logger.debug("Job script: \n %s" % self.job_script())
 
+    def _jobid_from_submit_output(self, out):
+        """ Get the unique identifier for a job from the submission status. """
+        return out.decode().split('.')[0].strip()
```

Comment:
This may not matter that much in practice but I would move `.decode()` to `JobQueueCluster.start_workers`. This will ensure that common functionality is in the base class.